pkg/operation/seed/seed.go

```diff
@@ -601,6 +606,45 @@ func BootstrapCluster(k8sGardenClient kubernetes.Interface, seed *Seed, config *
 		imageVectorOverwrites[name] = data
 	}
 
+	if gardenletfeatures.FeatureGate.Enabled(features.ManagedIstio) {
+		istiodImage, err := imageVector.FindImage(common.IstioIstiodImageName)
+		if err != nil {
+			return err
+		}
+
+		igwImage, err := imageVector.FindImage(common.IstioProxyImageName)
+		if err != nil {
+			return err
+		}
+
+		istioCRDs := istio.NewIstioCRD(chartApplier, "charts")
+		istiod := istio.NewIstiod(
+			&istio.IstiodValues{
+				TrustDomain: "cluster.local",
+				Image:       istiodImage.String(),
+			},
+			common.IstioNamespace,
+			chartApplier,
+			"charts",
+			k8sSeedClient.Client(),
+		)
+		igw := istio.NewIngressGateway(
+			&istio.IngressValues{
+				TrustDomain:     "cluster.local",
+				Image:           igwImage.String(),
+				IstiodNamespace: common.IstioNamespace,
+			},
+			common.IstioIngressGatewayNamespace,
+			chartApplier,
+			"charts",
+			k8sSeedClient.Client(),
+		)
+
+		if err := component.OpWaiter(istioCRDs, istiod, igw).Deploy(context.TODO()); err != nil {
+			return err
+		}
+	}
```

Comment:
cc @tim-ebert - this should probably be adapted to a dedicated `ManagedResource` as well after your PR is merged